### ISSUE TITLE
Can't pass "Room is now pending-name"

### ISSUE THREAD

**Issue by joao-p-marques** [matrix-org/matrix-appservice-slack#349]
I followed thew guide on the docs to bridge a room from slack, also tanking in consideration #348, but I always get the message "Room is now pending-name" after the command I send to @slackbot on my server.

Am I doing anything wrong or missing something?

**Comment by ryegye24** [comment]
I'm getting this same issue, did you ever figure out the problem @joao-p-marques ?

**Comment by joao-p-marques** [comment]
No, I ended up switching to the mx-puppet bridge, as that was a better fit for my use case and worked out of the box haha

It would be great to have this one working though for bringing an entire community to Matrix.

**Comment by pratikbalar** [comment]
+1

**Comment by pratikbalar** [comment]
https://github.com/matrix-org/matrix-appservice-slack/issues/659

**Comment by mkarg** [comment]
We have the the same problem. Would be cool if any of the committers could share some ideas.

**Comment by Radvendii** [comment]
Did anyone figure this out? I'm having the same problem.

**Comment by Radvendii** [comment]
Oh, I didn't notice the PR that just popped up. What timing! Thank you :pray: 

Does that mean this is just a cosmetic issue? Because I'm also not getting any messages forwarded.

---

Seems like it was indeed an unrelated issue.

**Comment by psrpinto** [comment]
I fixed this in https://github.com/matrix-org/matrix-appservice-slack/pull/756

Would be cool to have maintainers take a look at the PR, kinda crazy this issue has been there since 2020.
